REFERENCE.md

```diff
@@ -0,0 +1,90 @@
+---
+RFC: <PR number>
+Status: <Proposed>
+---
+
+# Title (Required)
+
+## Abstract (Required)
+
+A few sentences describing the feature.
+
+## Motivation (Required)
+
+What the feature solves and why the existing functionality is not enough.
+
+## Design considerations (Required)
+
+A description of the design constraints and requirements for the proposal. Comparisons with similar features in other projects.
+
+## Specification (Required)
+
+A more detailed description of the feature, including why certain details in the design have been chosen.
+
+### Commands (Optional)
+
+If any new commands are introduced:
+
+1. Command name
+   - **Request**
+   - **Response**
+
+### Authentication and Authorization (Optional)
+
+If there are any changes around authentication and authorization (ACL) access for commands/keyspace/channels.
```

Comment:
I would omit this, it seems very niche and not very likely to come up.

Reply:
Oh, I might have just been confused then. I would just add a description that this includes adding ACL categories as well. I thought you were referring to the overall auth system.